knative-operator/deploy/resources/quickstart/serverless-application-quickstart.yaml

```diff
@@ -83,7 +83,7 @@ spec:
       review:
         failedTaskHelp: >-
           This task isn’t verified yet. Try the task again, or [read
-          more](https://docs.openshift.com/container-platform/4.11/applications/odc-viewing-application-composition-using-topology-view.html#odc-scaling-application-pods-and-checking-builds-and-routes_viewing-application-composition-using-topology-view)
+          more](https://docs.openshift.com/container-platform/4.12/applications/odc-viewing-application-composition-using-topology-view.html#odc-scaling-application-pods-and-checking-builds-and-routes_viewing-application-composition-using-topology-view)
```

Comment:
If 4.12 is not out, the time we release, than stay 4.11 

My take - because otherwise: hard to browse 4.12 documentation, if not there with the 1.27 release